src/cli/cli.cpp

```diff
@@ -606,6 +609,50 @@ void Interpreter::ProcessDomainName(uint8_t aArgsLength, char *aArgs[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_CONFIG_DUA_ENABLE
+void Interpreter::ProcessDua(uint8_t aArgsLength, char *aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgsLength >= 1 && strcmp(aArgs[0], "iid") == 0, error = OT_ERROR_INVALID_COMMAND);
+
+    switch (aArgsLength)
+    {
+    case 1:
+    {
+        const otIp6InterfaceIdentifier *iid_ptr = otThreadGetFixedDuaInterfaceIdentifier(mInstance);
```

Comment:
```suggestion
        const otIp6InterfaceIdentifier *iid = otThreadGetFixedDuaInterfaceIdentifier(mInstance);
```

Reply:
it will shadow with below  `otIp6InterfaceIdentifier iid;`, prefer to keep as is.

Reply:
not shadow in current code structure now. will update.